crates/bevy_asset/src/handle.rs

```diff
@@ -58,12 +58,29 @@ impl HandleId {
 ///
 /// Handles contain a unique id that corresponds to a specific asset in the [Assets](crate::Assets)
 /// collection.
+///
+/// # Accessing the Asset
+///
+/// A handle is _not_ the asset itself, but should be seen as a pointer to the asset. Modifying a
+/// handle's `id` only modifies which asset is being pointed to. To get the actual asset, try using
+/// [`Assets::get`](crate::Assets::get) or [`Assets::get_mut`](crate::Assets::get_mut).
+///
+/// # Strong and Weak
+///
+/// A handle can be either "Strong" or "Weak". Simply put: Strong handles keep the asset loaded,
```

Comment:
Also, I commonly capitalized `Strong` and `Weak` when referring to those handle types. Not sure everyone will particularly enjoy that style...

If it's too distracting or not the preferred style, I can fix these to be lowercased!

Reply:
ok for me!